src/qt/glcanvas.cpp

```diff
@@ -64,6 +64,259 @@ void wxQtGLWidget::paintGL()
     EmitEvent(event);
 }
 
+// ----------------------------------------------------------------------------
+// wxGLContextAttrs: OpenGL rendering context attributes
+// ----------------------------------------------------------------------------
+// GLX specific values
+
+wxGLContextAttrs& wxGLContextAttrs::CoreProfile()
+{
+//    AddAttribBits(GLX_CONTEXT_PROFILE_MASK_ARB,
+//                  GLX_CONTEXT_CORE_PROFILE_BIT_ARB);
+    SetNeedsARB();
+    return *this;
+}
+
+wxGLContextAttrs& wxGLContextAttrs::MajorVersion(int val)
+{
+    if ( val > 0 )
+    {
+        if ( val >= 3 )
+            SetNeedsARB();
+    }
+    return *this;
+}
+
+wxGLContextAttrs& wxGLContextAttrs::MinorVersion(int val)
+{
+    if ( val >= 0 )
```

Comment:
...another empty `if()` block..
Mind that without it, there should probably be a `WXUNUSED()` around `val` to avoid "unused variable" compile warning. Or just delete its name (i.e. `wxGLContextAttrs::MinorVersion(int)`).
The same goes for other not used parameters.